index.bs

```diff
@@ -1483,6 +1561,34 @@ Whenever a [=WebTransport stream=] associated with a {{ReceiveStream}} |stream|
 
 </div>
 
+## `WebTransportReceiveStreamStats` Dictionary ##  {#receive-stream-stats}
+
+The <dfn dictionary>WebTransportReceiveStreamStats</dfn> dictionary includes
+information on stats specific to one {{WebTransportReceiveStream}}.
+
+<pre class="idl">
+dictionary WebTransportReceiveStreamStats {
+  DOMHighResTimeStamp timestamp;
+  unsigned long long amountReadFrom;
+  unsigned long long receivedProgress;
```

Comment:
bytesReceived ?

Is this contiguous bytes received, the total number of unique bytes received, or the total number of (non-unique) bytes received?